Exiled.API/Features/Roles/Scp106Role.cs

```diff
@@ -158,14 +158,20 @@ public bool IsStalking
         /// <summary>
         /// Forces SCP-106 to use its portal, and Teleport to position.
         /// </summary>
-        /// <param name="position"> where player will be Teleport.</param>
-        /// <param name="cost"> how mush Vigor it's required and conssume.</param>
+        /// <param name="position">Where the player will be Teleported.</param>
```

Comment:
```suggestion
        /// <param name="position">Where the player will be teleported.</param>
```
Teleported should not be capital in this case, it's not a proper noun nor the first word of a sentence.